client/driver/fabfile.py

```diff
@@ -132,6 +144,25 @@ def get_result():
     local(cmd)
 
 
+def _ready_to_start_controller():
+    file_path = '/home/shulij/ottertune/client/driver/oltp.log'
+    if not os.path.exists(file_path):
+        return False
+    else:
+        lines = open(file_path).readlines()
+        return len(lines) >= 15 and len(lines[14].split(' ')) > 2
+
+
+def _ready_to_shut_down_controller():
+    pid_file_path = '/home/shulij/ottertune/client/controller/pid.txt'
+    file_path = '/home/shulij/ottertune/client/driver/oltp.log'
+    if not os.path.exists(pid_file_path) or not os.path.exists(file_path):
+        return False
+    else:
+        lines = open(file_path).readlines()
+        return len(lines) >= 21 and len(lines[20].split(' ')) > 2
```

Comment:
Same here. Try to find the content in the log like 'output into file', instead of checking a fixed line.